staging/src/k8s.io/apiserver/pkg/authentication/authenticator/interfaces.go

```diff
@@ -29,6 +30,71 @@ type Token interface {
 	AuthenticateToken(ctx context.Context, token string) (*Response, bool, error)
 }
 
+// WrapInInstrumentedToken wraps the input Token in the InstrumentedRequest.
+func WrapInInstrumentedToken(token Token, authenticatorName string) *InstrumentedToken {
```

Comment:
These wrappers should be in a different package (or at the very least a different file).

Reply:
Moved into metrics.go file (in the same package).